tests/common/wifi/aws_test_wifi.c

```diff
@@ -1023,6 +1031,9 @@ TEST( Full_WiFi, AFQP_WIFI_GetHostIP_InvalidDomainName )
 
         /* Assert that the IP address is NOT found. */
         TEST_ASSERT_EQUAL_INT32( *( ( uint32_t * ) ucIPAddr ), 0 );
+    } else
```

Comment:
I now remember why I did not put a `else { TEST_FAIL(); }` here. It is because WIFI_GetHostIP( testwifiTEST_INVALID_DOMAIN_NAME, ucIPAddr ); might have an assert inside for us passing in an invalid domain. Then it was difficult to determine if that was indeed a failure case. I gave the libraries room to use an assert for invalid parameters and pass.